pbrEngine/engine.py

```diff
@@ -691,6 +749,291 @@ def _resetBlur(self):
+
+    def _enableCrystalMusic(self):
+        """
+        Enables the music for the Crystal Colosseum. (Battletopia10)
+        """
+        self._enableSong(Locations.SONG_CRYSTAL, "sound/Battletopia10.brstm")
+
+    def _enableLagoonMusic(self):
+        """
+        Enables the music for the Lagoon Colosseum.
+        """
+        self._enableSong(Locations.SONG_LAGOON, "sound/pokerev_music_demo_v2.brstm")
+
+    def _enableFanfareVar1(self):
+        """
+        Enables the first variation of the fanfare. (ME_Fan01)
+        """
+        self._enableSong(Locations.SONG_FANFARE_VAR1, "sound/ME_Fan01.brstm")
+
+    def _enableFanfareVar3(self):
+        """
+        Enables the third variation of the fanfare. (ME_Fan03)
+        """
+        self._enableSong(Locations.SONG_FANFARE_VAR3, "sound/ME_Fan03.brstm")
+
+    def _enableFanfareVar5(self):
+        """
+        Enables the fifth variation of the fanfare. (ME_Fan05)
+        """
+        self._enableSong(Locations.SONG_FANFARE_VAR5, "sound/ME_Fan05.brstm")
+
+    def _enableFanfareCompleted(self):
+        """
+        Enables the fanfare that plays when you have completed a colosseum. (ME_Fin)
+        """
+        self._enableSong(Locations.SONG_FANFARE_COMPLETED, "sound/ME_Fin.brstm")
+
+    def _enableColosseumSelectionMusic(self):
+        """
+        Enables the music for the Colosseum Selection screen. (System01)
+        """
+        self._enableSong(Locations.SONG_COLOSSEUM_SELECTION, "sound/System01.brstm")
+
+    def _enableReceptionMusic(self):
+        """
+        Enables the music for the Reception screen. (System03)
+        """
+        self._enableSong(Locations.SONG_RECEPTION, "sound/System03.brstm")
+
+    def _enableMonSelectionMusic(self):
+        """
+        Enables the music for the Pokemon Selection screen. (System05)
+        """
+        self._enableSong(Locations.SONG_MON_SELECTION, "sound/System05.brstm")
+
+    def _enableControlsMusic(self):
+        """
+        Enables the music for the Controls screen. (System09)
+        """
+        self._enableSong(Locations.SONG_CONTROLS, "sound/System09.brstm")
+
+    def _enableBossMusic(self):
+        path = ""
+        side = None
+        if self.avatars["blue"]["APPEARANCE"]["CHARACTER_STYLE"] > 6:
+            side = "blue"
+        elif self.avatars["red"]["APPEARANCE"]["CHARACTER_STYLE"] > 6:
+            side = "red"
+        if not side:
+            return
+        if self.avatars[side]["APPEARANCE"]["CHARACTER_STYLE"] in [7, 8]:
+            path = "sound/Battleboss01.brstm"
+        elif self.avatars[side]["APPEARANCE"]["CHARACTER_STYLE"] == 9:
+            path = "sound/Battleboss03.brstm"
+        elif self.avatars[side]["APPEARANCE"]["CHARACTER_STYLE"] == 10:
+            path = "sound/Battleboss04.brstm"
+        elif self.avatars[side]["APPEARANCE"]["CHARACTER_STYLE"] == 11:
+            path = "sound/Battleboss02.brstm"
+        elif self.avatars[side]["APPEARANCE"]["CHARACTER_STYLE"] == 12:
```

Comment:
more magic numbers everywhere

Reply:
you can extract stuff like `self.avatars[side]["APPEARANCE"]["CHARACTER_STYLE"]` to not have so many copy-paste accessors